doc/source/whatsnew/v0.21.0.txt

```diff
@@ -227,28 +227,30 @@ Other API Changes
 Deprecations
 ~~~~~~~~~~~~
 - :func:`read_excel()` has deprecated ``sheetname`` in favor of ``sheet_name`` for consistency with ``.to_excel()`` (:issue:`10559`).
+
+- ``pd.options.html.border`` has been deprecated in favor of ``pd.options.display.html.border`` (:issue:`15793`).
+
+Series.argmax and Series.argmin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Can you use `^^^` instead of `~~~` ? (then it becomes a subsection of the deprecations section)